Description: log changed

log changed, wtf
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 libsdl1.2 (1:1.2.15-7+2m7.1) unstable; urgency=medium
 .
   * Increment build number
Author: Jenkins Auto Builder <spam1@wizzup.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-08-04

--- /dev/null
+++ libsdl1.2-1.2.15/config.log
@@ -0,0 +1,279 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by configure, which was
+generated by GNU Autoconf 2.69.  Invocation command line was
+
+  $ ./configure --build=arm-linux-gnueabihf --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libdir=${prefix}/lib/arm-linux-gnueabihf --libexecdir=${prefix}/lib/arm-linux-gnueabihf --runstatedir=/run --disable-rpath --enable-sdl-dlopen --disable-video-ggi --disable-video-svga --disable-video-aalib --disable-nas --disable-esd --disable-arts --disable-alsa-shared --disable-pulseaudio-shared --disable-x11-shared --enable-video-caca
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = ml1
+uname -m = armv7l
+uname -r = 5.7.0
+uname -s = Linux
+uname -v = #1 SMP PREEMPT Sun Jul 12 18:58:38 UTC 2020
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/local/bin
+PATH: /usr/bin
+PATH: /bin
+PATH: /usr/local/games
+PATH: /usr/games
+PATH: /usr/local/sbin
+PATH: /usr/sbin
+PATH: /sbin
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2881: checking for gcc
+configure:2897: found /usr/bin/gcc
+configure:2908: result: gcc
+configure:3137: checking for C compiler version
+configure:3146: gcc --version >&5
+ccache: error: Could not find compiler "gcc" in PATH
+configure:3157: $? = 1
+configure:3146: gcc -v >&5
+ccache: error: Could not find compiler "gcc" in PATH
+configure:3157: $? = 1
+configure:3146: gcc -V >&5
+ccache: error: Could not find compiler "gcc" in PATH
+configure:3157: $? = 1
+configure:3146: gcc -qversion >&5
+ccache: error: Could not find compiler "gcc" in PATH
+configure:3157: $? = 1
+configure:3177: checking whether the C compiler works
+configure:3199: gcc    conftest.c  >&5
+ccache: error: Could not find compiler "gcc" in PATH
+configure:3203: $? = 1
+configure:3241: result: no
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| /* end confdefs.h.  */
+| 
+| int
+| main ()
+| {
+| 
+|   ;
+|   return 0;
+| }
+configure:3246: error: in `/pr/sdlstuf/libsdl1.2-1.2.15':
+configure:3248: error: C compiler cannot create executables
+See `config.log' for more details
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=
+ac_cv_env_CFLAGS_value=
+ac_cv_env_CPPFLAGS_set=
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=
+ac_cv_env_CXXFLAGS_value=
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_LDFLAGS_set=
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_LT_SYS_LIBRARY_PATH_set=
+ac_cv_env_LT_SYS_LIBRARY_PATH_value=
+ac_cv_env_XMKMF_set=
+ac_cv_env_XMKMF_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=arm-linux-gnueabihf
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_prog_ac_ct_CC=gcc
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ALLOCA=''
+ALSA_CFLAGS=''
+ALSA_LIBS=''
+AR=''
+ARTSCONFIG=''
+AS=''
+AWK=''
+BUILD_CFLAGS=''
+BUILD_LDFLAGS=''
+CACACONFIG=''
+CC='gcc'
+CFLAGS=''
+CPP=''
+CPPFLAGS=''
+CXX=''
+CXXCPP=''
+CXXFLAGS=''
+DEFS=''
+DIRECTFBCONFIG=''
+DLLTOOL=''
+DSYMUTIL=''
+DUMPBIN=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP=''
+ENABLE_SHARED_FALSE=''
+ENABLE_SHARED_TRUE=''
+ENABLE_STATIC_FALSE=''
+ENABLE_STATIC_TRUE=''
+ESD_CFLAGS=''
+ESD_CONFIG=''
+ESD_LIBS=''
+EXEEXT=''
+EXTRA_CFLAGS=''
+EXTRA_LDFLAGS=''
+FGREP=''
+GREP=''
+INCLUDE=''
+INSTALL_DATA=''
+INSTALL_PROGRAM=''
+INSTALL_SCRIPT=''
+LD=''
+LDFLAGS=''
+LIBOBJS=''
+LIBS=''
+LIBTOOL=''
+LIPO=''
+LN_S=''
+LTLIBOBJS=''
+LT_AGE=''
+LT_CURRENT=''
+LT_RELEASE=''
+LT_REVISION=''
+LT_SYS_LIBRARY_PATH=''
+MANIFEST_TOOL=''
+NASM=''
+NASMFLAGS=''
+NM=''
+NMEDIT=''
+OBJDUMP=''
+OBJECTS=''
+OBJEXT=''
+OSMESA_CONFIG=''
+OTOOL64=''
+OTOOL=''
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME=''
+PACKAGE_STRING=''
+PACKAGE_TARNAME=''
+PACKAGE_URL=''
+PACKAGE_VERSION=''
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+POW_LIB=''
+PTH_CONFIG=''
+RANLIB=''
+SDLMAIN_LDFLAGS=''
+SDLMAIN_OBJECTS=''
+SDLMAIN_SOURCES=''
+SDL_BINARY_AGE=''
+SDL_CFLAGS=''
+SDL_INTERFACE_AGE=''
+SDL_LIBS=''
+SDL_MAJOR_VERSION=''
+SDL_MICRO_VERSION=''
+SDL_MINOR_VERSION=''
+SDL_RLD_FLAGS=''
+SDL_STATIC_LIBS=''
+SDL_VERSION=''
+SED=''
+SET_MAKE=''
+SHELL='/bin/bash'
+SOURCES=''
+STRIP=''
+WINDRES=''
+XMKMF=''
+X_CFLAGS=''
+X_EXTRA_LIBS=''
+X_LIBS=''
+X_PRE_LIBS=''
+ac_aux_dir=''
+ac_ct_AR=''
+ac_ct_CC='gcc'
+ac_ct_CXX=''
+ac_ct_DUMPBIN=''
+bindir='${exec_prefix}/bin'
+build='arm-linux-gnueabihf'
+build_alias='arm-linux-gnueabihf'
+build_cpu=''
+build_os=''
+build_vendor=''
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host=''
+host_alias=''
+host_cpu=''
+host_os=''
+host_vendor=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+libdir='${prefix}/lib/arm-linux-gnueabihf'
+libexecdir='${prefix}/lib/arm-linux-gnueabihf'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+runstatedir='/run'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME ""
+#define PACKAGE_TARNAME ""
+#define PACKAGE_VERSION ""
+#define PACKAGE_STRING ""
+#define PACKAGE_BUGREPORT ""
+#define PACKAGE_URL ""
+
+configure: exit 77
